<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rtuFile_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input rtu File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rtu_file_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="csvTags_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Tags csv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_tags_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="start_time_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="StartTime_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time M/D/YYYY HH:MM AM/PM</string>
    </property>
   </widget>
   <widget class="QLabel" name="EndTime_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End Time  M/D/YYYY HH:MM AM/PM</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="end_time_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ReportName_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Report Name (defaults to report)</string>
    </property>
   </widget>
   <widget class="QLabel" name="csvFile_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>csv File Name (defaults to csvFile)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Inputs_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Required Inputs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Desired Outputs:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reliability_check_1_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.1: Readings within Expected Range of Operation</string>
    </property>
   </widget>
   <widget class="QLabel" name="Reliability_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reliability Checks</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reliability_check_2_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.2: Measurment Units were Verified</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reliability_check_3_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.3: Quality of the Signals is GOOD in the rtu File</string>
    </property>
   </widget>
   <widget class="QLabel" name="TC_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timeliness and Completeness Checks</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tc_check_1_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2.1: Points are Updated on a Frequent Enough Basis in the rtu File</string>
    </property>
   </widget>
   <widget class="QLabel" name="Robustness_Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Robustness Checks</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="robustness_check_1_check_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>680</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4.1: Signals are Stable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_push_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>770</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_push_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>770</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rtu_file_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="csv_tags_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="csv_file_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Input csv File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="report_name_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Input Report Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reliability_check_4_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.4: Quality of the Signals is GOOD in the review File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="partial_commissioning_check_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Partial Commissioning</string>
    </property>
   </widget>
   <widget class="QLabel" name="reviewFile_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Review File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="review_file_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="review_file_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="tc_check_2_check_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2.2: Points are Updated on a Frequent Enough Basis in the review File</string>
    </property>
   </widget>
   <widget class="QLabel" name="FLAT_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLAT Threshold (defaults to 5 min)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FLAT_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Input FLAT Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="operating_flowrate_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min/Max Operating Flowrates m3/hr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minimum_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Minimum</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maximum_line_edit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QLabel" name="Accuracy_Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Accuracy Checks</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="accuracy_check_1_check_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3.1: Digital/Analog Signals are in Close Agreement</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="accuracy_check_3_check_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3.3: Flow Readings are in Close Agreement with Other Measuremnets</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="accuracy_check_2_check_box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3.2: Acceptable Signal-to-Noise Ratio</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Required Inputs</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>1. rtu File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>2. CSV with all tags to assess</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>3. Review File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>4. Start Time (of analysis)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>5. End Time (of analysis)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>6. Minimum Operating Flowrate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>7. Maximum Operating Flowrate</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="full_acceptance_check_box">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Acceptance</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy_range_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy Range (default to 1%)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accuracy_range_line_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Range of Accuracy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSingle_Variant">
    <property name="title">
     <string>Main</string>
    </property>
   </widget>
   <addaction name="menuSingle_Variant"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
